src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -77,7 +76,9 @@ const CityColumnChart = (props) => {
 
   // We only want to show the short description as the chart title
   // if the chart is not stacked
-  const yAxisTitle = isStacked ? {} : { text: shortDescription };
+  const yAxisTitle = isMultiple ? {} : { text: shortDescription };
+
+  const ceiling = props.ceiling || (props.stacked ? 1 : null);
```

Comment:
Ceiling can probably go entirely. It's only purpose was to manually pass in 1 for stacked charts. As is, it's never actually passed in to the component.

Reply:
It is passed in, though - line 117

Reply:
I was unclear :) I mean a value for `props.ceiling` is never passed in to `<CityColumnChart>`.